src/Modera/MjrIntegrationBundle/DependencyInjection/Configuration.php

```diff
@@ -122,6 +122,9 @@ public function getConfigTreeBuilder()
                 ->scalarNode('app_base_class')
                     ->defaultValue('')
                 ->end()
+                ->scalarNode('moment_js_version')
+                    ->defaultValue(null)
```

Comment:
Probably would make sense to specify a default version here instead of JsResourcesProvider, this way dump:config would show it as well and thus require no further digging.